coalib/bearlib/languages/definitions/VisualBasic.py

```diff
@@ -0,0 +1,11 @@
+from coalib.bearlib.languages.Language import Language
+
+
+@Language
+class VisualBasic:
+    aliases = 'vb',
+    extensions = '.vb', '.bas'
+    comment_delimiter = "'"
+    string_delimiters = {'"': '"'}
+    multiline_string_delimiters = {'"': '"'}
```

Comment:
Add a comment above this that multiline strings are only possible from Visual Basic 14.0 and above.

and then `multiline_string_delimiters = string_delimiters`

(and .. not worth even thinking about now , but .. omg .. https://github.com/coala/coala/issues/5587 )